vmdb/lib/tasks/evm.rake

```diff
@@ -36,8 +36,9 @@ namespace :evm do
     EvmApplication.update_stop
   end
 
-  task :compile_assets => :environment do
```

Comment:
joev: are you sure you want to remove the dependency on the environment?

response: yes, the environment is loaded via `assets:precompile`, so it was not needed